shared_model/validators/settings.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_SETTINGS_HPP
+#define IROHA_SHARED_MODEL_SETTINGS_HPP
+
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
+
+  namespace validation {
+
+    /**
+     * Structure that holds configurable ledger settings
+     */
+    struct Settings {
+      explicit Settings(size_t max_description_size = 64);
+
+      size_t max_description_size;
+
+      static const shared_model::interface::types::SettingKeyType
+          kMaxDescriptionSizeKey;
```

Comment:
this does not seem a part of `shared_model::validatin::Settings`. please move it somewhere to `iroha::ametsuchi`. maybe somewhere near `PostgresSettingQuery`.

Reply:
How to relate to https://github.com/hyperledger/iroha/pull/133#discussion_r316501315?

Reply:
Fixed